tests/conftest.py

```diff
@@ -0,0 +1,270 @@
+import sys
+from gdcapi.auth import roles
+import os
+from signpost import Signpost
+from multiprocessing import Process
+from gdcdatamodel.models import Edge, Node
+from gdcapi.auth import AuthDriver
+from psqlgraph import PsqlGraphDriver
+import pytest
+import time
+from gdcapi.api import app as _app, app_init
+from mock import patch
+import requests
+from userapi import app as userapi_app
+from userapi import app_init as userapi_app_init
+from userapi import utils
+from cdis_oauth2client import OAuth2Client
+from userdatamodel import Base
+
+
+from elasticsearch import Elasticsearch
+
+from gdcapi.test_settings import GDC_ES_INDEX as INDEX
+from gdcapi.test_settings import PSQL_USER_DB_CONNECTION
+from gdcapi.test_settings import Fernet, HMAC_ENCRYPTION_KEY
+from userdatamodel import models as usermd
+from userdatamodel import Base as usermd_base
+from userdatamodel.driver import SQLAlchemyDriver
+from cdisutilstest.code.storage_client_mock import get_client
+import os
+import json
+
+from gdcapi.config import LEGACY_MODE
+
+here = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, here)
+
+
+class UserapiTestSettings(object):
+    from boto.s3.connection import OrdinaryCallingFormat
```

Comment:
is it possible to import those fixtures from gdcapi instead of duplicating the code?